mule-extensions-api/src/main/java/org/mule/runtime/extension/api/declaration/type/ExtensionObjectTypeHandler.java

```diff
@@ -45,6 +46,15 @@
   private final LiteralTypeAnnotation literalTypeAnnotation = new LiteralTypeAnnotation();
   private final TypedValueTypeAnnotation typedValueTypeAnnotation = new TypedValueTypeAnnotation();
 
+  private final Map<Class<?>, ParsingContext> wrappedTypesContextProvider = new HashMap<Class<?>, ParsingContext>() {
```

Comment:
wrappedTypesContextProvider -> wrappedTypesContexts

Also , this map should be unmodifiable